package.json

```diff
@@ -64,11 +63,16 @@
     "eslint": "3.6.0",
     "font-awesome": "4.6.3",
     "handlebars": "4.0.5",
+    "imports-loader": "0.6.5",
```

Comment:
I don't think this is even possible or reasonable. Unless I am mistaken, I believe @nornagon is talking about real libs (like jquery-ui) and not our "libs" in `client/js/libs`.

Or maybe there is a different/better way to do so, I don't know. I think this is good enough for a first shot at it and we can improve later.